deps/key_keycode/key_to_keycode_win.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
Please consider to update the old tests ('input_xxx_basic') with this new change, otherwise we would have two duplicated pieces of code, which is not good for maintenance.

Reply:
ok, will update it~